sklearn/cluster/_affinity_propagation.py

```diff
@@ -271,6 +285,14 @@ class AffinityPropagation(ClusterMixin, BaseEstimator):
         ``euclidean`` are supported. 'euclidean' uses the
         negative squared euclidean distance between points.
 
+    random_state : int or np.random.RandomStateInstance, default: None
```

Comment:
default isn't None now.

Reply:
I think we document the effective default, not the actual one, so in this case it would be 0, not `warn`.

Reply:
defaulted to `0`